documentation/1.3/tour/modules.md

```diff
@@ -18,23 +18,23 @@ We're going to learn about *packages* and *modules*.
 ## Packages and imports
 
 Every source file&mdash;along with everything declared in the 
-file&mdash;belongs to a package. Even source files that don't belong to a 
-named package are considered to belong to the "default" package. It's the 
-location of a source file within a source directory that determines which 
-package a source file belongs to.
+file&mdash;belongs to a package. Declarations from a different package
+have to be imported before they can be used in a source file.
 
 ### Source files and packages
 
 There's no `package` statement in Ceylon source files. The compiler determines 
 the package and module to which a toplevel program element belongs by the 
-location of the source file in which it is declared. For example, if `source` 
-is a source directory, and if a class named `Hello` is defined in the file 
-`source/org/jboss/hello/Hello.ceylon`, then `Hello` belongs to the package 
-`org.jboss.hello`.
+directory path of the source file in which it is declared. The file name itself 
+has no significance to the compiler, but conventionally follows the content 
+of the source file.
```

Comment:
How is "the name of the source file itself is not significant" harder to understand than "the file name itself has no significance to the compiler, but conventionally follows the content of the source file"?